docs/build/guides/conventions/working-with-contract-specs.mdx

```diff
@@ -0,0 +1,1264 @@
+---
+title: Work with contract specs in Java, Python, and PHP
+description: A guide to understanding and interacting with Soroban smart contracts in different programming languages.
+hide_table_of_contents: false
```

Comment:
This line can just be removed entirely. it defaults to `false`, so there's no need to specify that.